src/main/java/net/fabricmc/loom/extension/LoomGradleExtensionApiImpl.java

```diff
@@ -128,7 +128,7 @@ protected LoomGradleExtensionApiImpl(Project project, LoomFiles directories) {
 				.convention(project.provider(() -> !isForge()));
 		this.modProvidedJavadoc.finalizeValueOnRead();
 		this.intermediary = project.getObjects().property(String.class)
-				.convention("https://maven.fabricmc.net/net/fabricmc/intermediary/%1$s/intermediary-%1$s-v2.jar");
+				.convention(true ? "https://maven.legacyfabric.net/net/legacyfabric/v2/intermediary/%1$s/intermediary-%1$s-v2.jar" : "https://maven.fabricmc.net/net/fabricmc/intermediary/%1$s/intermediary-%1$s-v2.jar");
```

Comment:
We are not going to depend on legacy fabric. Loom has an option to not use intermediary at all